com.github.PintaProject.Pinta.yaml

```diff
@@ -1,14 +1,14 @@
 app-id: com.github.PintaProject.Pinta
 runtime: org.gnome.Platform
-runtime-version: '3.38'
+runtime-version: '40'
 sdk: org.gnome.Sdk
 command: pinta
 rename-appdata-file: pinta.appdata.xml
 rename-desktop-file: pinta.desktop
 rename-icon: pinta
 copy-icon: true
 sdk-extensions:
-  - org.freedesktop.Sdk.Extension.mono6
+  - org.freedesktop.Sdk.Extension.dotnet5
 finish-args:
   # X11 + XShm access
```

Comment:
if you upgrade to Gtk3, it's probably a good idea to add `--socket=wayland` and replace `--socket=x11` to `--socket=fallback-x11`.

Reply:
Thanks for the hint! I updated it accordingly.